indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -260,6 +262,11 @@
                                context concept :variable-association)]
     (get-elastic-version-with-associations context concept tag-associations variable-associations)))
 
+(defmethod get-elastic-version :variable
```

Comment:
I'm like the changes like this your making, moving us towards thinking more in terms of a consistent, internal API.